<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StepDataWidget</class>
 <widget class="QWidget" name="StepDataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string comment="@title">Set step data</string>
  </property>
  <property name="whatsThis">
   <string comment="@info:whatsthis">&lt;para&gt;Set the id and the text of the step&lt;/para&gt;
&lt;para&gt;The text is shown to the user when the tutorial changes to that step.&lt;/para&gt;
&lt;para&gt;On the other hand, the id is only used internally and never shown to the user. It is used to identify the steps in the tutorial when it has to change from one to step to another.&lt;/para&gt;</string>
  </property>
  <layout class="QVBoxLayout" name="StepDataWidgetLayout">
   <item>
    <widget class="QGroupBox" name="stepDataGroupBox">
     <property name="title">
      <string comment="@title">Step data</string>
     </property>
     <layout class="QVBoxLayout" name="stepDataGroupBoxLayout">
      <item>
       <layout class="QHBoxLayout" name="idLayout">
        <item>
         <widget class="QLabel" name="idLabel">
          <property name="text">
           <string comment="@label:textbox">Id:</string>
          </property>
          <property name="buddy">
           <cstring>idLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KLineEdit" name="idLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="textLayout">
        <item>
         <widget class="QLabel" name="textLabel">
          <property name="text">
           <string comment="@label:textbox">Text:</string>
          </property>
          <property name="buddy">
           <cstring>textTextEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KTextEdit" name="textTextEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
